src/components/AccordionList.jsx

```diff
@@ -5,6 +5,7 @@ import AccordionDetails from "@mui/material/AccordionDetails";
 import Typography from "@mui/material/Typography";
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
 import { FormattedMessage } from "react-intl";
+
```

Comment:
:pick: remove this line

Reply:
Addressed:  f1c51260db64b8bc3578cbe88ac1fc5b035cc865